lib/index.js

```diff
@@ -254,17 +248,11 @@ function gcloud(config) {
     if (scopedApis[apiName]) {
       gcloudExposedApi[apiName] = new Class(config);
     } else {
-      gcloudExposedApi[apiName] = function(options) {
-        return new Class(util.extendGlobalConfig(config, options));
-      };
+      gcloudExposedApi[apiName] = Class;
     }
 
-    // Extend each API to pick up static members, e.g. Datastore.int and
-    // Storage.acl.
-    extend(gcloudExposedApi[apiName], Class);
-
     return gcloudExposedApi;
-  }, {});
+  }, { config_: config });
```

Comment:
Can you add a `/*! Developer Documentation` block for this function to explain how this part works?